visualization.shadertoy/platforms.txt

```diff
@@ -0,0 +1 @@
+all
```

Comment:
Glew is opengl. Ios is gles, so no.
29. okt. 2015 10:49 skrev "Wolfgang Schupp" <notifications@github.com>:

> In visualization.shadertoy/platforms.txt
> <https://github.com/xbmc/repo-binary-addons/pull/12#discussion_r43365665>:
>
> > @@ -0,0 +1 @@
> > +all
>
> glew depends for ios are missing in shadertoy
> @notspiff <https://github.com/notspiff> is it supposed to work there?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/repo-binary-addons/pull/12/files#r43365665>.
>